src/Migrations/20200203143917-create-minig-rewards-table.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
                 type: sequelize.STRING
             },
             rewardAmount: {
-                type: sequelize.STRING
+                type: sequelize.NUMERIC(200, 100)
```

Comment:
We won't store decimals so don't need the second argument (scale). I think that the max FILL supply is 2B (10^12) and then extra 18 decimal points representation. So I'd put 30 or 32 to be on the safe side.

I am not expert on this one but it's probably better to have it more precise to real numbers because of calculations and storage.